mail-client/bower/bower-0.9.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A curses terminal client for the Notmuch email system"
+HOMEPAGE="https://github.com/wangp/bower"
+SRC_URI="https://github.com/wangp/bower/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="app-crypt/gpgme
+	>=dev-lang/mercury-11.07"
+RDEPEND="net-mail/notmuch
+	sys-apps/coreutils
```

Comment:
Why do you need to dep on coreutils?

Reply:
Bower uses `base64` and that binary is found in sys-apps/coreutils. See [the documentation](https://github.com/wangp/bower/blob/master/README.md), section "requirements".

Reply:
Ok, I guess that's 'not standard enough' to justify direct dep.

/me wonders what would provide 'base64' on FreeBSD systems.